packages/app/src/pages/SwapPage/SwapComponent.tsx

```diff
@@ -124,6 +126,13 @@ export function SwapComponent({
     setTyping(false);
   }, [previewValue]);
 
+  useEffect(() => {
+    if (hasSwapped) {
```

Comment:
You can use a atom here, get the reference from the setTypeing

Reply:
+1